app/helpers/template_helpers.py

```diff
@@ -54,6 +54,13 @@ def __init__(
             "EQ_GOOGLE_TAG_MANAGER_AUTH"
         )
         self._survey_type = cookie_session.get("theme")
+        self._preview_questions = False
+        if (metadata := get_metadata(current_user)) and (
+            metadata.schema_url or metadata.schema_name
+        ):
+            self._preview_questions = load_schema_from_metadata(
+                metadata=metadata, language_code=language
+            ).json.get("preview_questions", False)
```

Comment:
Pass this in instead of loading the schema here. Where ContextHelper is initialised, we already need the schema.

Reply:
Ok, found a schema where we initialise ContextHelper so extracting from `survey_config` now..